code/game/machinery/doors/firedoor.dm

```diff
@@ -25,6 +25,8 @@
 	var/boltslocked = TRUE
 	var/list/affecting_areas
 	var/being_held_open = FALSE
+	var/knocksound = 'sound/effects/Glassknock.ogg'
+	var/bashsound = 'sound/effects/Glassbash.ogg'
```

Comment:
These sound effects don't exist. Does this actually work? (Insert comments about windows here)

Reply:
What do you mean they don't exist? The files are in the code.
https://github.com/tgstation/tgstation/blob/master/sound/effects/glassbash.ogg
https://github.com/tgstation/tgstation/blob/master/sound/effects/glassknock.ogg

I tested it in game and it works.

Reply:
```suggestion
	var/knock_sound = 'sound/effects/Glassknock.ogg'
	var/bash_sound = 'sound/effects/Glassbash.ogg'
```